mantinone/core-algorithms
# Core Algorithms, with Exotic Godwit! Goal #123

Team name: #exotic-godwit
Authors: @lumodon & @mantinone

Our project contains several algorithms and one Chrome extension!
To test the js algorithms, run `npm test`.
To test the c++ algorithm, run `npm run cpp`
To test the python algorithm, run `npm run python`
To test our Chrome Extension, go into your chrome extensions page and check the `Developer Mode` box in the upper-right.  Then click the `Load unpacked extension` button and pick the `./extension` folder.  The extension should appear in your list of extensions.  Click the icon in your navbar to select the style of text enhancement you'd like to see, then load up any webpage with lots of text to test it out!
